frontend/src/components/widget/Lookup/RawLookup.js

```diff
@@ -335,7 +336,10 @@ export class RawLookup extends Component {
       typeaheadSupplier,
     } = this.props;
 
-    const inputValue = this.inputSearch.value;
+    let inputValue = this.inputSearch.value;
+    if (inputValue.trim() === '') {
+      inputValue = ' ';
+    }
```

Comment:
why if the input value is blank then we are setting it to ' ' (one space)?

PS: nevertheless, note that u might get an error in case inputValue is null

Reply:
to send a space character (' ') as the query when the input is empty. This way, clicking on the input field will trigger the same typeahead results as typing a space.